packages/webservice/app/build.gradle.kts

```diff
@@ -48,20 +58,18 @@ tasks.withType<KotlinCompile> {
 }
 
 tasks.register<NpmTask>("installUIDependencies") {
-    setArgs(listOf("ci"))
+    setArgs(listOf("install"))
```

Comment:
Can we revert this? `ci` mode allows use to install via the package-lock.json

Reply:
this was one of the problems i encountered while testing. The package-lock.json went out of sync and prevented the packages being installed/updated. we need something to  keep it updated/sync. Is there something already in place for it ?

Reply:
reverted